### DISCUSSION TITLE
Typescript error on linting

### DISCUSSION THREAD

**Discussion by richardhowarth** [react-hook-form/react-hook-form#2215] [Q&A]
Hi, 

can anyone help please,
I am getting the following error on linting, it refers to the getValue method of useForm but I am not using this anywhere.
I would appreciate some direction please on how to remove this error

Thanks in advance 
Richard

```
[tslint-plugin] Starting linter in separate process...
Error at node_modules\react-hook-form\dist\types\form.d.ts:200:130: Type 'TFieldName' cannot be used to index type 'UnpackNestedValue<TFieldValues>'.

```

**Comment by phzimmermann** [comment]
i updated typescript from 3.7.3 to 3.9.7.

**Comment by mzvast** [comment]
ERROR in /Users/apple/@Projects/vr-officialwebsite/node_modules/react-hook-form/dist/types/form.d.ts(134,130):
TS2536: Type 'TFieldName' cannot be used to index type 'UnpackNestedValue<TFieldValues>'.
Version: typescript 3.5.3
Time: 420ms

**Reply by bluebill1049** [reply]
pretty sure we on ^4.0.0

**Reply by mzvast** [reply]
> pretty sure we on ^4.0.0

I find out a way to ignore this error.
add 

```javascript
...
{
  "compilerOptions": {
   
    "skipLibCheck": true // ADD THIS LINE
  },
 ...
}

```
